### PR TITLE
VR-4852: Add run.download_deployment_crd()

### PR DESCRIPTION
**Pull request by convoliution**

### CODE REVIEW

File: client/verta/tests/test_deployment.py [hunk 1]
```diff
@@ -1025,3 +1021,31 @@ def test_undeployed_get_error(self, experiment_run, model_for_deployment):
...
+            model_deployment, config_map = yaml.safe_load_all(f)
+
+        assert model_deployment['kind'] == "ModelDeployment"
+        assert model_deployment['metadata']['name'] == experiment_run.id
+
+        assert config_map['kind'] == "ConfigMap"
+        assert config_map['metadata']['name'] == "model--{}".format(experiment_run.id)
+
+        model_api = json.loads(config_map['data']['model_api.json'])
+        assert model_api == model_for_deployment['model_api'].to_dict()
```
Comment (convoliution) [inline client/verta/tests/test_deployment.py:1051]: @conradoverta Is there anything else I should cover in this test?

Comment (conradoverta) [reply client/verta/tests/test_deployment.py:1051]: No. We'll have to remember to change the test as things goes.

File: client/verta/verta/client.py [hunk 2]
```diff
@@ -4066,6 +4066,60 @@ def get_deployed_model(self):
         status = self.get_deployment_status()
         return deployment.DeployedModel.from_url(status['url'], status['token'])
 
+    def download_deployment_crd(self, download_to_path, path=None, token=None, no_token=False):
```
Comment (conradoverta) [inline client/verta/verta/client.py]: Crd => yaml. Let's not have the user worry about details.

**Review by conradoverta — COMMENTED** [review]
Very small changes.

File: client/verta/verta/client.py (hunk 2 above)
Comment (convoliution) [reply client/verta/verta/client.py]: @conradoverta I'm not sure what "crd => yaml" is conveying.
Is the suggestion to rename the method to `download_deployment_yaml`? Or automatically append ".yaml" to the destination filename? Or to return a dict object from reading the YAML?

Comment (conradoverta) [reply client/verta/verta/client.py]: Rename

**Pull request merged**